pkg/dockerregistry/server/configuration/configuration.go

```diff
@@ -0,0 +1,140 @@
+package configuration
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/docker/distribution/configuration"
+)
+
+var (
+	// CurrentVersion is the most recent Version that can be parsed.
+	CurrentVersion = configuration.MajorMinorVersion(1, 0)
+
+	ErrUnsupportedVersion = errors.New("Unsupported openshift configuration version")
```

Comment:
I'd keep it lowercase since it refers to the `openshift` section in the config file. There was an effort to get rid of product names in our source codes. That's why I'd rather keep it pointing to the section name, not the product name.